chat-components/src/common/Constants.ts

```diff
@@ -51,4 +51,152 @@ export enum ElementType {
     CurrentCallEndCallButton = "CurrentCallEndCallButton",
     Utility = "Utility",
     Custom = "Custom"
-}
\ No newline at end of file
+}
+
+export const Ids = class {
```

Comment:
I think it's a good time to formalize these Ids before any customers start using them heavily. The Ids today are mixed up, some starting with oc-lcw, some not.

For chat-component layer, let's use camelCase without "-" for all Ids. For example, currentCallTimer instead of oc-lcw-CurrentCall-timer

Reply:
Double check if they are used in any selector-based logics, otherwise we'll have hard-to-find bugs

Reply:
fixed